bpf-gpl/tc.c

```diff
@@ -436,7 +437,7 @@ static CALI_BPF_INLINE int calico_tc(struct __sk_buff *skb)
 	goto finalize;
 }
 
-__attribute__((section("1/1")))
+SEC("classifier/1")
```

Comment:
Iirc the 1/1 convention was for `tc` to be able to load it. Since we are loading it ourselves now, can we have more meaningful section names now? Like `tc/main` or `tc/icmp` etc. ?